projects/ion/src/lib/step/step.component.spec.ts

```diff
@@ -0,0 +1,139 @@
+import { Component } from '@angular/core';
+import { TestBed } from '@angular/core/testing';
+import { FormsModule } from '@angular/forms';
+import { render, screen } from '@testing-library/angular';
+import { IonIconComponent } from '../icon/icon.component';
+import { StepComponent, StepType } from './step.component';
+
+const defaultValue: StepType[] = [
+  {
+    label: 'Step 1',
+  },
+  {
+    label: 'Step 2',
+  },
+  {
+    label: 'Step 3',
+  },
+];
+
+describe('Static StepComponent', () => {
+  const sut = async (
+    customProps: StepType[] = defaultValue
+  ): Promise<HTMLElement> => {
+    await render(StepComponent, {
+      componentProperties: {
+        steps: customProps,
+      },
+      declarations: [IonIconComponent],
+      imports: [FormsModule],
+    });
+    return screen.findByTestId('ion-step');
+  };
+
+  it('should render step component with 3 steps', async () => {
+    await sut();
+    expect(screen.getByText('Step 3')).toBeTruthy();
+  });
+  it('should render step component with 3 steps, first checked', async () => {
+    await sut([
+      {
+        label: 'Step 1',
+        status: 'checked',
+      },
+      {
+        label: 'Step 2',
+      },
+      {
+        label: 'Step 3',
+      },
+    ]);
+    expect(
+      screen.getByText('Step 3') && screen.getByTestId('step-1-checked')
+    ).toBeTruthy();
+  });
+  it('should render step component with 3 steps, first checked and second with error and description', async () => {
+    await sut([
+      {
+        label: 'Step 1',
+        status: 'checked',
+      },
+      {
+        label: 'Step 2',
+        status: 'error',
+        description: 'Error',
+      },
+      {
+        label: 'Step 3',
+      },
+    ]);
+    expect(
+      screen.getByText('Step 3') &&
+        screen.getByTestId('step-1-checked') &&
+        screen.getByTestId('step-2-error') &&
+        screen.getByTestId('description-2')
+    ).toBeTruthy();
```

Comment:
```suggestion
    expect(screen.getByTestId('step-1-checked')).toBeTruthy();
    expect(screen.getByTestId('step-2-error')).toBeTruthy();
```

do not multiple conditionals in a single expect, you too can improve this with toHaveClass.